### PR TITLE
Fix autoprecomp test by adding build after develop

### PR DESCRIPTION
**Pull request by ianshmean**
Fixes the test breakage introduced by #2197

### CODE REVIEW

**Comment by DilumAluthge** [comment]
It will be good to merge this as soon as CI is all green.

**Comment by KristofferC** [comment]
Sorry, I merged the revert. Could you add the unrevert as the first commit in this PR?

**Comment by ianshmean** [comment]
Precompile tests are passing now. Failures seem artifact-related

**Comment by DilumAluthge** [comment]
@ianshmean Can you rebase on the latest master? The artifacts tests should be fixed now on master.

**Comment by KristofferC** [comment]
Aren't CI running on the merge commit?

**Comment by ianshmean** [comment]
Yeah, evidently they are. Just needed a retrigger

File: test/api.jl [hunk 1]
```diff
@@ -8,7 +8,7 @@ import Pkg.Types.PkgError, Pkg.Resolve.ResolverError
 using UUIDs
 
 using ..Utils
-
+original_path = pwd()
 @testset "Pkg.activate" begin
     temp_pkg_dir() do project_path
         cd_tempdir() do tmp
```
Comment (KristofferC) [inline test/api.jl:14]: The `cd_tempdir()` shoudl already do this I think.

Comment (KristofferC) [reply test/api.jl:14]: What changes the active directory in these tests?

Comment (fredrikekre) [reply test/api.jl:14]: xref https://github.com/JuliaLang/Pkg.jl/pull/2089 which was never fixed

Comment (ianshmean) [reply test/api.jl:14]: I wouldn't mind if someone more knowledgeable took this PR over

**Comment by ianshmean** [comment]
Something in here is `rm`-ing the working directory. If I add a `@show pwd()` immediately before and after, the one after fails
https://github.com/ianshmean/Pkg.jl/blob/d7d42ddebc706314761ea961642c82f635ff2711/test/api.jl#L79-L101

**Comment by ianshmean** [comment]
Should be good now?

**Pull request merged**